webapi/src/main/scala/org/knora/webapi/slice/search/api/SearchEndpoints.scala

```diff
@@ -298,6 +316,16 @@ final case class SearchRestService(
     response     <- renderer.render(searchResult, opts)
   } yield response
 
+  def incomingLinksSearch(
```

Comment:
the route is called "searchIncomingLinks", so maybe this method should/could be called the same?